apps/leo_gateway/priv/leo_gateway.conf

```diff
@@ -109,6 +109,13 @@ bucket_prop_sync_interval = 300
 ## wtperf is set to the same value.
 ## nfs.nfsd.wtmax = 5242880
 
+## NFSd's Readdir Entry Table Scan Interval (leo-project/leofs#612)
+## nfs.nfsd.readdir_scan_int = 1800
+
+## NFSd's Readdir Entry TTL (leo-project/leofs#612)
```

Comment:
IMHO,
Timeout that can affect the latency of online tasks like HTTP/RPC would be desirable to be set by as much fine grained as possible OTOH offline ones like this would be sufficient to be set by coarse gained one (second in this case).